### PR TITLE
[16.0][MIG] delivery_package_number: Migration to 16.0 

### PR DESCRIPTION
**Pull request by chienandalu**
Standard migration

cc @Tecnativa TT40420

please review @victoralmau @pedrobaeza

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]
Code review OK

File: delivery_package_number/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Picking Package Number",
+    "summary": "Set or compute number of packages for a picking",
+    "version": "16.0.1.1.0",
```
Comment (pedrobaeza) [inline delivery_package_number/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

Comment (chienandalu) [reply delivery_package_number/__manifest__.py]: Oops. Fixed :sweat_smile:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**